Infrastructure.md

```diff
@@ -0,0 +1,58 @@
+# Infrastructure Setup
+
+The provisioning system depends on a central master, hosted at http://classroom.puppet.com
```

Comment:
Well, and cloning the repo and sharing secrets is much harder to do with zero requirements. But doable.